scripts/build_image

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+
+# set readable variables
+export JAVA_VERSION_TEMPLATE="$1"
+export NODE_VERSION_TEMPLATE="$2"
+export MBT_VERSION="$3"
+
+# set jvm and node version in Dockerfile
+if [ "$JAVA_VERSION_TEMPLATE" = "11" ]
```

Comment:
This `if/else` seems quite verbose and "repeating itself" in the three branches.
Can you simply calculate `JAVA_VERSION` directly from `JAVA_VERSION_TEMPLATE`?
Should you?

Reply:
Do you mean change the code like this?

export JAVA_VERSION="-java${JAVA_VERSION_TEMPLATE}"
if [ "$JAVA_VERSION_TEMPLATE" = "11" ]
then
	................
elif [ "$JAVA_VERSION_TEMPLATE" = "8" ]
then
	....................
elif [ "$JAVA_VERSION_TEMPLATE" = "17" ]
then
	......................
else
	........................
fi